<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Plotter" name="widgetPlotter" native="true">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>10</y>
      <width>591</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>471</width>
      <height>431</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCV">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCS">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCE">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXY">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>XY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonYZ">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>YZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXZ">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>XZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSV">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SV</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>210</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Ellipsoid</string>
    </property>
    <widget class="QSlider" name="horizontalSliderRX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderRY">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderRZ">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Sphere</string>
    </property>
    <widget class="QSlider" name="horizontalSliderRD">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>490</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>ColorBox</string>
    </property>
    <widget class="QSlider" name="horizontalSliderRed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderGreen">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderBlue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderAlpha">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonRotClk">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↻</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRotCClk">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↺</string>
    </property>
   </widget>
   <widget class="ColorWindow" name="widgetColor" native="true">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>500</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInv">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Inv</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>420</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Slice</string>
    </property>
    <widget class="QSlider" name="horizontalSliderSlice">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Box Size</string>
    </property>
    <widget class="QSlider" name="horizontalSliderX">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderY">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderZ">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Radius Settings</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>View File</string>
    </property>
    <widget class="QPushButton" name="pushButtonVECT">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>VECT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOFF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GridSet">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SizeX">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SizeY">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SizeZ">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>groupBox_7</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>widgetPlotter</zorder>
   <zorder>pushButtonPV</zorder>
   <zorder>pushButtonCV</zorder>
   <zorder>pushButtonPB</zorder>
   <zorder>pushButtonCB</zorder>
   <zorder>pushButtonPS</zorder>
   <zorder>pushButtonCS</zorder>
   <zorder>pushButtonPE</zorder>
   <zorder>pushButtonCE</zorder>
   <zorder>pushButtonXY</zorder>
   <zorder>pushButtonYZ</zorder>
   <zorder>pushButtonXZ</zorder>
   <zorder>pushButtonSV</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>pushButtonRotClk</zorder>
   <zorder>pushButtonRotCClk</zorder>
   <zorder>widgetColor</zorder>
   <zorder>pushButtonInv</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_10</zorder>
   <zorder>GridSet</zorder>
   <zorder>SizeX</zorder>
   <zorder>SizeY</zorder>
   <zorder>SizeZ</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <signal>moveX(int)</signal>
    <signal>moveY(int)</signal>
    <signal>planeChosen(int)</signal>
    <slot>mudaFrequencia(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ColorWindow</class>
   <extends>QWidget</extends>
   <header>colorwindow.h</header>
   <container>1</container>
   <slots>
    <slot>changeRed()</slot>
    <slot>changeGreen()</slot>
    <slot>changeBlue()</slot>
    <slot>changeAlpha()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>929</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>948</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>773</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>929</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>919</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSlice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>900</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>899</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
